### PR TITLE
Post: Useful documentation - ADRs

### PR DESCRIPTION
**Pull request by sharnik**
### Description

New blog post on using ADRs in React Native projects.

### Demo

https://deploy-preview-35--brains-and-beards-com.netlify.app/blog/2023-useful-documentation-adrs/

### CODE REVIEW

File: content/blog/2023-useful-documentation-adrs/index.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Finally, it saves onboarding time. Often, when a new member joins a team they might wonder why something is built a particular way, or try to port over their favourite solutions from previous projects. Having clear ADRs that explain your patterns and choices helps new hires quickly learn the reasoning behind them and see whether there's a better solution that they could offer.
+
+
+## I'm sold, how would I want to document that?
+
+First, setup a place for your ADRs, probably somewhere in the `docs` folder in your repo. You have a `docs` folder, right?
+
+Second, ADRs are time-bound, meaning you've made a decision at a particular point in time and it's important to track that. Of course, you can always check the date in the document, but I find it helpful to number the filenames. For example: `1-i18n-library.md`, `2-state-management-provider.md`, etc.
+
+Third, decide on a format. You can [pick one of the popular ones/(https://github.com/joelparkerhenderson/architecture-decision-record/tree/main/templates) or write your own. It doesn't matter that much which one you pick, but having _a_ structure will make it easier to write them.
```
Comment (PoorTable) [inline content/blog/2023-useful-documentation-adrs/index.mdx]: Small formating issue 
```suggestion
Third, decide on a format. You can [pick one of the popular ones](https://github.com/joelparkerhenderson/architecture-decision-record/tree/main/templates) or write your own. It doesn't matter that much which one you pick, but having _a_ structure will make it easier to write them.
```

Comment (sharnik) [reply content/blog/2023-useful-documentation-adrs/index.mdx]: Thanks!

File: content/blog/2023-useful-documentation-adrs/index.mdx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Finally, it saves onboarding time. Often, when a new member joins a team they might wonder why something is built a particular way, or try to port over their favourite solutions from previous projects. Having clear ADRs that explain your patterns and choices helps new hires quickly learn the reasoning behind them and see whether there's a better solution that they could offer.
+
+
+## I'm sold, how would I want to document that?
+
+First, setup a place for your ADRs, probably somewhere in the `docs` folder in your repo. You have a `docs` folder, right?
+
+Second, ADRs are time-bound, meaning you've made a decision at a particular point in time and it's important to track that. Of course, you can always check the date in the document, but I find it helpful to number the filenames. For example: `1-i18n-library.md`, `2-state-management-provider.md`, etc.
+
+Third, decide on a format. You can [pick one of the popular ones](https://github.com/joelparkerhenderson/architecture-decision-record/tree/main/templates) or write your own. It doesn't matter that much which one you pick, but having _a_ structure will make it easier to write them.
```
Comment (marras) [inline content/blog/2023-useful-documentation-adrs/index.mdx]: Link broken, did you mean https://github.com/joelparkerhenderson/architecture-decision-record/tree/main/locales/en/templates ?

Comment (sharnik) [reply content/blog/2023-useful-documentation-adrs/index.mdx]: They move the files around overnight 🤦‍♂️
Thanks!

**Pull request merged**